package.json

```diff
@@ -84,8 +84,8 @@
     "start-prod": "cross-env NODE_ENV=production SHOW_ALL_DAOS=false NETWORK=xdai node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js  --config webpack.dev.config.js",
     "start-staging-kovan": "cross-env NODE_ENV=production SHOW_ALL_DAOS=true NETWORK=kovan node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
     "start-staging-rinkeby": "cross-env NODE_ENV=production SHOW_ALL_DAOS=true NETWORK=rinkeby node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
+    "start-staging-xdai": "cross-env NODE_ENV=production SHOW_ALL_DAOS=true NETWORK=xdai node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
```

Comment:
This change is in order to be consistent with `start-staging-kovan` and `start-staging-rinkeby`?

Reply:
yes, and with alchemy 1.0